lib/vm.js

```diff
@@ -17,27 +17,25 @@ const realRunInThisContext = Script.prototype.runInThisContext;
 const realRunInContext = Script.prototype.runInContext;
 
 Script.prototype.runInThisContext = function(options) {
-  if (options && options.breakOnSigint) {
-    return sigintHandlersWrap(() => {
-      return realRunInThisContext.call(this, options);
-    });
-  } else {
-    return realRunInThisContext.call(this, options);
-  }
+  let breakOnSigint = options && options.breakOnSigint;
+  let runInThisContext = () => realRunInThisContext.call(this, options);
+
+  if (breakOnSigint) return sigintHandlersWrap(runInThisContext);
+
+  return runInThisContext();
 };
 
 Script.prototype.runInContext = function(contextifiedSandbox, options) {
-  if (options && options.breakOnSigint) {
-    return sigintHandlersWrap(() => {
-      return realRunInContext.call(this, contextifiedSandbox, options);
-    });
-  } else {
-    return realRunInContext.call(this, contextifiedSandbox, options);
-  }
+  let breakOnSigint = options && options.breakOnSigint;
+  let runInContext = () => realRunInContext.call(this, contextifiedSandbox, options);
```

Comment:
You are right I didn't imagine that it was so sensible,

I will rollback these and only submit the `Change legacy var to let` commit,

Wdyt?